src/routes/Videos.js

```
@@ -37,9 +39,72 @@ export default class Videos extends Component {
       })
   }
 
+
+
+  tagStringToArray = str => {
+   const tagsArr = str.split(', ').filter(Boolean);
+   if (tagsArr.length > 3) {
+     tagsArr.length = 3;
```

This won't stop the tagsArray from being great than 3.  If it is greater than 3, you need to either throw an error that says you can only have 3 tags OR you can grab the first 3 tags in the array and call it a day.